src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -2309,6 +2309,10 @@ https://api.livechatinc.com/v3.4/agent/action/list_routing_statuses \
 
 ### Mark Events as Seen
 
+It marks an agent’s events up to a specific time as read on a customer’s chat.
+
+This method requires an `access_token` to authorize requests. More information about Agent authorization flows can be found [here](/authorization/authorizing-api-calls#agent-authorization-flows).
```

Comment:
I was not sure if this information is necessary here. There is also the same method for a customer, so it might be confusing if someone wants to check both methods at the same time. However, at the same time, it should be rather clear that if you call the API on behalf of an agent you should simply use 'access_token'.